HouseRules_Essentials/Rules/StartCardsModifiedRule.cs

```diff
@@ -47,6 +47,50 @@ private static void Patch(Harmony harmony)
                 postfix: new HarmonyMethod(
                     typeof(StartCardsModifiedRule),
                     nameof(Piece_CreatePiece_Postfix)));
+            harmony.Patch(
+                original: AccessTools.Method(typeof(Inventory), "RestoreReplenishables"),
+                prefix: new HarmonyMethod(
+                    typeof(StartCardsModifiedRule),
+                    nameof(Inventory_RestoreReplenishables_Prefix)));
+        }
+
+        private static bool Inventory_RestoreReplenishables_Prefix(bool __result, Piece piece)
+        {
+            if (!_isActivated)
+            {
+                return true;
+            }
+
+            __result = false;
+            for (int i = 0; i < piece.inventory.Items.Count; i++)
+            {
+                Inventory.Item value = piece.inventory.Items[i];
+                var targetRefresh = (value.flags & 224) >> 5;
+                var countdown = (value.flags & 28) >> 2;
```

Comment:
nit: shifting first and then applying a simpler mask (7, or binary literal `0b111`), or applying mask without shifting (e.g., `0b11100`) may be easier to read.